lib/pansophy/config_synchronizer.rb

```diff
@@ -0,0 +1,34 @@
+require 'pansophy'
+require 'facets/kernel/blank'
+
+class ConfigSynchronizer
+  include Memoizable
+
+  def pull
+    verify_app_name!
+    Pansophy.pull(app_name, remote_path, local_path, overwrite: true)
+  end
+
+  private
+
+  def app_name
+    ENV['AUDITOR_APP_NAME']
+  end
+  memoize :app_name
+
+  def remote_path
+    Pathname.new('config').join(version.to_s)
+  end
+
+  def version
+    ENV['CONFIG_VERSION'] || '1.0'
+  end
```

Comment:
Yes, I was planning on having this in Elastic beanstalk's env, so if we do a new config we can change it there, e.g. say we want to switch mail servers without deploying.

Currently we have to edit the sync task on the project, change config then deploy,